lib/bridge/MatrixHandler.js

```diff
@@ -17,6 +17,9 @@ const MSG_PMS_DISABLED = "[Bridge] Sorry, PMs are disabled on this bridge.";
 const MSG_PMS_DISABLED_FEDERATION = "[Bridge] Sorry, PMs are disabled on " +
 "this bridge over federation.";
 
+const KICK_RETRY_DELAY_MS = 15000;
```

Comment:
trying to kick every 15s feels a bit aggressive
and 500ms of jitter is nothing
i'd suggest 30+/-15 secs or something